spec/requests/providers/other_assets_spec.rb

```diff
@@ -0,0 +1,120 @@
+require 'rails_helper'
+
+RSpec.describe 'provider other assets requests', type: :request do
+  let(:application) { create :application, :with_applicant }
+  let(:oad) { application.create_other_assets_declaration! }
+  let(:application_id) { application.id }
+
+  before { get providers_legal_aid_application_other_assets_path(application) }
+
+  describe 'GET citizens/own_home' do
```

Comment:
Change the describe  from Citizen to Provider i.e  `Get Provider providers/applications/:legal_aid_application_id/own_home`